ydb/docs/en/core/concepts/federated_query/greenplum.md

```diff
@@ -0,0 +1,76 @@
+# Working with Greenplum databases
+
+This section provides basic information on working with external [Greenplum](https://greenplum.org) databases.
+
+To work with an external Greenplum database, you need to follow these steps:
+1. Create a [secret](../datamodel/secrets.md) containing the password for connecting to the database.
+    ```sql
+    CREATE OBJECT greenplum_datasource_user_password (TYPE SECRET) WITH (value = "<password>");
+    ```
+1. Create an [external data source](../datamodel/external_data_source.md) that describes a specific database within the Greenplum cluster. In the `LOCATION` parameter you need to pass the network address of [master node](https://www.greenplumdba.com/greenplum-master) Greenplum. By default, the [namespace](https://docs.vmware.com/en/VMware-Greenplum/6/greenplum-database/ref_guide-system_catalogs-pg_namespace.html) `public` is used for reading, but this value can be changed using the optional `SCHEMA` parameter. The network connection is made using the standard ([Frontend/Backend Protocol](https://www.postgresql.org/docs/current/protocol.html)) over TCP transport (`PROTOCOL="NATIVE"`). You can enable encryption of connections to the external database using the `USE_TLS="TRUE"` parameter.
```

Comment:
It's better to stick with links to somewhat official resources, and greenplumdba.com doesn't look like one. I'd recommend replacing them.